hugo/content/contribute.md

```diff
@@ -3,83 +3,279 @@ title: Contribute
+If you find any issues, take a look at [Report Bugs](#report-bugs-and-feature-requests).
 
-# Promotion
+<!-- mention documentation? Potential problem with TOC linking as the categories have the same name -->
+<!-- 
+### Documentation
+
+See [below](#documentation). -->
 
-Get the word out. This means more than writing about it on your blog ;)
+### Mumble theme
 
-* Ask (voice-)server hosters on if they also provide Mumble servers if you can’t find them.
-* Coordinate with one of the hosters providing free mumble servers and start promoting Mumble as the voicechat of choice for your favorite game.
-* Add mumble hosters you found to the {{< wiki Hosters />}} wiki page / update information.
-* Create How-Tos, Screencasts, Tutorials or Benchmarks on typical issues and topics, and publish them on prominent platforms.
-* Spread the word in groups – join our [Steam Group](https://steamcommunity.com/groups/Mumble_VoIP)
+<!-- Add more details, descriptions etc.? -->
+You find the Mumble theme in it's own [repository on Github](https://github.com/mumble-voip/mumble-theme).
 
-# Skinning
+#### Skins
 
 Mumble skins bring variety into Mumbles design and choices for different tastes.
 
 Create new skins for Mumble! Information can be found on our {{< wiki Skinning "Skinning wiki page" />}} and existing skins we know about on the {{< wiki Skins "Skins wiki page" />}}.
 
-# Coding
 
-## 3rd party Interfaces
+# Documentation & Website
 
-Web interfaces, web-viewers, administration programs, CMS website modules – all neat things we would love to see! These could be written in any language supported by our Ice based RPC Interface (using the deprecated {{< wiki DBus />}} Interface is no longer advised). You could start your own project or link up with an {{< wiki "3rd Party Applications" "already existing one" />}}. To get an idea about how interaction with Murmur might look like you can take a look at the example scripts in our git repositories script folder.
+Improve our Documentation and Website.
 
-## Ice authenticators
+## Documentation
 
-See {{< wiki Authenticators />}}. We’re happy for every authenticator and authenticator improvement you and the community can bring us!
+<!-- Todo: 
 
-## Positional Audio
+Add a concrete plan:
+- what documentation we want to have
+- what should be converted from the wiki
+- what is done & what is not
 
-Most of our positional audio plugins regularly break on game updates. To read on how you can help us update the plugins see the {{< wiki Pluginguide />}} page. You can even fix the update-issue forever for open source games, or if you want to add positional audio to your game, see {{< wiki Link />}}.
+- add issues and open pull requests
 
-## Mumble itself
+-->
+
+We plan to move all our documentation to the Website and Github Repo respectively.
```

Comment:
```suggestion
We plan to move all our documentation to the Website (enduser documentation) and GitHub repo (developer documentation) respectively.
```

Reply:
Well, essentially that is not correct imo.
1. All documentation is moved to github, because the website is on github. (I know nitpicking :stuck_out_tongue_winking_eye: ).
2. As far as I have seen and understood, we want:
     - enduser documentation on github and website
     - developer documentation on github and website

Reply:
1. Fair enough ^^
2. We certainly want the enduser docs on the website. The developer docs idk. Personally I think it's fine to have the developer info only in the main repo. The important keyword here is _main repo_ though. Dev docs should be were the devs are and that's the source code. This makes putting it on the website a bit tedious though (I think) as it's in the wrong repo

Reply:
I think all of it can (and should) be on the website as well and the discussion between @Kissaki and @davidebeatrici in #35 indicates they want to implement a development documentation as well.

A completely different topic is the question of the repo.
Right now we have most of it in the main repo (Mumble) and it could stay this way (because it can "simply" be converted and implemented on the website), but there are two arguments against it:
If we would have a seperate documentation repo, we could:
- keep the download size and visible structure minimal (though there are methods like git sparse-checkout for it, but that might not be known to everyone)
- keep better track of issues and pull requests specific to documentation

Reply:
> Dev docs should be were the devs are and that's the source code.

This is really important to me. I don't mind putting the docs on the website _in addition_ to that as long as we find a solution that does not require (manual) duplication of the docs.
I could imagine a process in which the CI in this repo pulls the main repo and includes the developer docs in this way. Alternatively we could probably use something like GitHub Actions to automatically update a mirror of the dev docs in this repo. Thus the dev docs here would be automatically generated from the actual docs in the main repo.

Reply:
Sounds good to me.